### PR TITLE
Add range_fields parameter to support querying on date type with range filter

### PR DESCRIPTION
**Pull request by yuku-t**
Make it possible to search with "created_at:<2015-04-16"

review plz @r7kamura

### CODE REVIEW

File: lib/qiita/elasticsearch/token.rb [hunk 1]
```diff
@@ -55,6 +57,35 @@ def positive?
...
+      #   range_query #=> "2015-04-16"
+      def range_query
+        range_match[:query]
+      end
+
+      private
+
+      def range_match
+        @range_match = RANGE_TERM_REGEXP.match(@term) || {}
+      end
```
Comment (r7kamura) [inline lib/qiita/elasticsearch/token.rb]: You might want to cache the result of `Regexp#match` by memoizing like so:

```rb
@range_match ||= ...
```

**Comment by r7kamura** [comment]
Found wrong words "renge" at some points :weary:

**Comment by r7kamura** [comment]
Otherwise looks fine :+1:

**Comment by yuku-t** [comment]
Fixed where you pointed out. Thx

**Pull request merged**